### DISCUSSION TITLE
whats the point of..

### DISCUSSION THREAD

**Discussion by bitministry** [RehanSaeed/Schema.NET#17] [Q&A]
having multiple OneOrMany<string,SomeComplex> Width
but single value Category ?

**Comment by RehanSaeed** [comment]
Not sure I completely understand.

Schema.org lets you include N number of items for each property. It's a very loose standard. This is what `OneOrMany` lets you do but limits it to certain concrete types as defined by schema.org.

**Comment by RehanSaeed** [comment]
Feel free to reopen if you have more to add.
